addons/account/wizard/account_move_reversal.py

```diff
@@ -26,14 +26,50 @@ class AccountMoveReversal(models.TransientModel):
             ('modify', 'Full refund and new draft invoice')
         ], string='Credit Method', required=True,
         help='Choose how you want to credit this invoice. You cannot "modify" nor "cancel" if the invoice is already reconciled.')
-    journal_id = fields.Many2one('account.journal', string='Use Specific Journal', help='If empty, uses the journal of the journal entry to be reversed.', check_company=True)
+    journal_id = fields.Many2one(
+        comodel_name='account.journal',
+        string='Use Specific Journal',
+        required=True,
+        compute='_compute_journal_id',
+        readonly=False,
+        store=True,
+        check_company=True,
+        help='If empty, uses the journal of the journal entry to be reversed.',
+    )
     company_id = fields.Many2one('res.company', required=True, readonly=True)
+    available_journal_ids = fields.Many2many('account.journal', compute='_compute_available_journal_ids')
 
     # computed fields
     residual = fields.Monetary(compute="_compute_from_moves")
     currency_id = fields.Many2one('res.currency', compute="_compute_from_moves")
     move_type = fields.Char(compute="_compute_from_moves")
 
+    @api.depends('move_ids')
+    def _compute_journal_id(self):
+        for record in self:
+            if record.journal_id:
+                record.journal_id = record.journal_id
```

Comment:
:suspect:

Reply:
Was told to always attribute value in _compute